sdk/core/azure-core/src/main/java/com/azure/core/http/policy/UserAgentPolicy.java

```diff
@@ -62,12 +62,12 @@ public UserAgentPolicy(String userAgent) {
      * @param sdkVersion Version of the client library.
      * @param configuration Configuration store that will be checked for the AZURE_TELEMETRY_DISABLED.
      */
-    public UserAgentPolicy(String sdkName, String sdkVersion, Configuration configuration) {
+    public UserAgentPolicy(String sdkName, String sdkVersion, Configuration configuration, String versionString) {
```

Comment:
With a `ServiceVersion` interface, you can then have this `UserAgentPolicy` take the `ServiceVersion` directly.